Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleShapeView.m

```diff
@@ -176,15 +178,24 @@ - (void)setCenter:(CGPoint)center
     [self updateLayers];
 }
 
+- (void)setBubbleView:(nullable OWSBubbleView *)bubbleView
+{
+    _bubbleView = bubbleView;
+
+    [self updateLayers];
+}
+
 - (void)updateLayers
 {
     if (!self.shapeLayer) {
         return;
     }
-
     if (!self.bubbleView) {
         return;
     }
+    if (!self.isConfigured) {
```

Comment:
We don't want to do this work until the view is configured.  It's an optimization but also avoids configuring for the wrong mode.